### PR TITLE
Regen

### PR DESCRIPTION
**Pull request by damccorm**

### CODE REVIEW

**Comment by stephenmichaelf** [comment]
Was this off a build from a fully deployed version of the API?

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "azure-devops-node-api",
     "description": "Node client for Azure DevOps and TFS REST APIs",
-    "version": "9.0.3",
+    "version": "9.1.0",
```
Comment (stephenmichaelf) [inline package.json]: Major version bump?

File: samples/package-lock.json [hunk 2]
```diff
@@ -7,32 +7,9 @@
     "azure-devops-node-api": {
```
Comment (stephenmichaelf) [inline samples/package-lock.json]: Is this supposed to be changed?

File: test/package-lock.json [hunk 3]
```diff
@@ -7,27 +7,9 @@
     "azure-devops-node-api": {
```
Comment (stephenmichaelf) [inline test/package-lock.json]: Is this supposed to be changed?

File: api/interfaces/CommentsInterfaces.ts [hunk 4]
```diff
@@ -0,0 +1,457 @@
+﻿/*
```
Comment (stephenmichaelf) [inline api/interfaces/CommentsInterfaces.ts]: Where did we get this file from? Looks new.

Comment (damccorm) [reply api/interfaces/CommentsInterfaces.ts]: It is new, it was necessary for the build to succeed though, some WIT files were depending on it

File: test/package-lock.json (hunk 3 above)
Comment (damccorm) [reply test/package-lock.json]: Yeah, it should've been before probably. It now matches the package.json

File: samples/package-lock.json (hunk 2 above)
Comment (damccorm) [reply samples/package-lock.json]: Yeah, it should've been before probably. It now matches the package.json

File: package.json (hunk 1 above)
Comment (damccorm) [reply package.json]: I'm never sure whether we should go major or minor. In theory this shouldn't be breaking, right? Idk, we need to figure out a good plan here. I'm fine bumping to major for now, I'll do that

**Comment by damccorm** [comment]
> Was this off a build from a fully deployed version of the API?

Its from a clean build of master - I don't think deploying should affect anything, right?

File: package.json (hunk 1 above)
Comment (damccorm) [reply package.json]: Done

**Comment by stephenmichaelf** [comment]
The code in VSO master hasn't rolled so it could have generated code against APIs that a user can't hit yet.

**Comment by damccorm** [comment]
> The code in VSO master hasn't rolled so it could have generated code against APIs that a user can't hit yet.

Closing in favor of #368

**Pull request closed without merging**